amethyst_renderer/src/screen_space.rs

```diff
@@ -0,0 +1,11 @@
+use amethyst_core::specs::{Component, NullStorage};
+
+/// Indicates that the entity on which this is placed should use the coordinates of the window instead of the coordinates relative to the `Camera`.
+/// Mostly used with user interface elements.
+/// Do not use with entities located outside of -1000 <= z < 1000.
```

Comment:
re:magic numbers, it's difficult for people to discover this -- "you don't know until you know" type of thing, which is a gotcha. 

also, is -1000 because camera_z = 1000, and view distance 2000? this makes me prefer having 2 passes (second pass ignores depth)

Reply:
You can't ignore the depth though.

Reply:
haha, "make it configurable". I can live with that, can you update this sentence then:

```rust
/// Do not use with entities located outside of the visible z range as defined by
/// `ScreenSpaceSettings`.
```